src/components/Table/TableBody/RowActionsCell/RowActionsCell.jsx

```diff
@@ -240,7 +240,9 @@ class RowActionsCell extends React.Component {
                           action.renderIcon ? (
                             <OverflowMenuContent title={action.labelText}>
                               {typeof action.renderIcon === 'string' ? (
-                                <Icon icon={action.renderIcon} description={action.labelText} />
+                                React.createElement(icons[action.renderIcon], {
+                                  'aria-label': action.labelText,
```

Comment:
Was changing from from 'description' to 'aria-label' deliberate?

Reply:
Yeah, just mirroring what happens when using `<Icon/>`, `description` is used as the `aria-label`

[Link to carbon-components-react `<Icon/>` implementation](https://github.com/carbon-design-system/carbon/blob/f7fca2be6b552c330a4e3e1341e9056ddde50790/packages/react/src/components/Icon/Icon.js#L146)